CHANGES.md

```diff
@@ -10,6 +10,9 @@
  - fix hover position computation on the presence of Utf characters
    (@ejgallego, #597, thanks to Pierre Courtieu for the report and
    example, closes #594)
+ - workaround seemingly VSCode activation bug that prevented extension
+   activation (@ejgallego @r3m0t, #598, cc #596, reported by Théo
+   Zimmerman)
```

Comment:
You should probably edit the changelog entry if this is not a VS Code bug and the fix here is the proper fix as opposed to a workaround.

Reply:
Indeed, thanks for pointing it out!